ctapipe/io/containers.py

```diff
@@ -78,6 +77,7 @@ class RawCameraContainer(Container):
                             "integrated ADC data (n_pixels)"))
     adc_samples = Item(Map(), ("map of channel to arrays of "
                                "(n_pixels, n_samples)"))
+    num_samples = Item(int, "number of time samples")
```

Comment:
Understood...but with the way num_samples is returned from pyhessio, it does change per event. If a telescope has data in the current event, this could equal 96, but for the next event if the telescope does not have data, this would then equal 0 (but the key in event.dl0 would not exist of course).